.github/workflows/release-docker.yml

```
@@ -15,12 +21,18 @@ jobs:
     permissions:
       contents: read
       packages: write
+    strategy:
+      matrix:
+        include:
+          - bin: maker
+          - bin: taker
```

```suggestion
        bin: [maker, taker]
```

`include` is designed for specifying combinations manually. If all you need is permutations, there is a simpler way.

---

thanks :)